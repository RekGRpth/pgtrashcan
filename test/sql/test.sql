CREATE TABLE test1 (a int, b text);
CREATE TABLE test2 (a int, b text);

DROP TABLE test1;

\dt public.*
\dt "Trash".*

DROP TABLE test2;

\dt public.*
\dt "Trash".*

DROP TABLE "Trash".test1;

\dt "Trash".*

DROP SCHEMA "Trash" CASCADE;

\dn *rash*
