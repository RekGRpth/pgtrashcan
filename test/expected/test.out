CREATE TABLE test1 (a int, b text);
CREATE TABLE test2 (a int, b text);
DROP TABLE test1;
\dt public.*
         List of relations
 Schema | Name  | Type  |  Owner   
--------+-------+-------+----------
 public | test2 | table | postgres
(1 row)

\dt "Trash".*
         List of relations
 Schema | Name  | Type  |  Owner   
--------+-------+-------+----------
 Trash  | test1 | table | postgres
(1 row)

DROP TABLE test2;
\dt public.*
      List of relations
 Schema | Name | Type | Owner 
--------+------+------+-------
(0 rows)

\dt "Trash".*
         List of relations
 Schema | Name  | Type  |  Owner   
--------+-------+-------+----------
 Trash  | test1 | table | postgres
 Trash  | test2 | table | postgres
(2 rows)

DROP TABLE "Trash".test1;
\dt "Trash".*
         List of relations
 Schema | Name  | Type  |  Owner   
--------+-------+-------+----------
 Trash  | test2 | table | postgres
(1 row)

DROP SCHEMA "Trash" CASCADE;
NOTICE:  drop cascades to table "Trash".test2
\dn *rash*
List of schemas
 Name | Owner 
------+-------
(0 rows)

